frontend/src/components/Displays/LocalizationVerification/ScheduleMissionChecklistDialog.tsx

```diff
@@ -0,0 +1,256 @@
+import { Button, Checkbox, Dialog, Typography } from '@equinor/eds-core-react'
+import { DeckMapView } from 'utils/DeckMapView'
+import { HorizontalContent, StyledDialog, VerticalContent } from './ScheduleMissionStyles'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { Robot } from 'models/Robot'
+import { ChangeEvent, useState } from 'react'
+import { useInstallationContext } from 'components/Contexts/InstallationContext'
+
+interface ScheduleMissionChecklistDialogProps {
+    closeDialog: () => void
+    scheduleMissions: () => void
+    robot: Robot
+    missionDeckName: string
+}
+
+interface CheckDialogProps {
+    setIsCheckConfirmed: (b: boolean) => void
+    robot: Robot
+    deckName: string
+}
+
+const LocalisationDialog = ({ setIsCheckConfirmed, robot, deckName }: CheckDialogProps) => {
+    const { TranslateText } = useLanguageContext()
+    const { installationDecks } = useInstallationContext()
+
+    const newDeck = installationDecks.find((deck) => deck.deckName === deckName)
+
+    return (
+        <>
+            <Dialog.Header>
+                <Typography variant="h5">{TranslateText('Confirm placement of robot')}</Typography>
+            </Dialog.Header>
+            <Dialog.Content>
+                <VerticalContent>
+                    <Typography>
+                        {`${robot.name} (${robot.model.type}) ${TranslateText(
+                            'needs to be placed on marked position on'
```

Comment:
I want to change this to must but think that's scope creep and i can do that another time. Also below the english can be sinplified and more straight to the point

Reply:
This text is copied from how it was before, so I hope it's okay to leave it and the other rewordings for another PR?

Reply:
Yeah yeah, that's what i was thinking don't worry about it, just commented so i can find this code later when we work on the next PR